README.md

```diff
@@ -215,6 +215,16 @@ options:
   # commonly created by using the "Update branch" button in the UI.
   ignore_update_merges: false
 
+  # If present, commits authored and committed by users meeting the conditions
+  # are ignored for the purposes of approval. This means the users will not
+  # count as contributors and their commits will not invalidate approval if
+  # invalidate_on_push is enabled. Both the author and the committer must match
+  # the conditions to ignore the commit.
```

Comment:
Should we also add the caveat from your pr description?
```
Note that commit authorship is easy to forge, so
this is not a secure option unless paired with a pre-receive hook that
enforces accurate commit attribution.
```

Reply:
Yes, but I think I want to add a "Security" section to the README that discusses this, overwriting status checks, commit editing, and some of the other issues/limitations, then add a note here referencing the new section.